src/Tribe/Main.php

```diff
@@ -99,6 +99,24 @@ public function __construct( $context = null ) {
 		add_action( 'tribe_common_loaded', [ $this, 'tribe_common_app_store' ], 10 );
 		add_action( 'customize_controls_print_styles', [ $this, 'load_tec_variables' ], 10 );
 
+		add_action(
+			'current_screen',
+			function () {
+				/**
+				 * Fires just before headers are sent.
+				 *
+				 * We can use this action instead of headers_sent().
+				 *
+				 * Especially where a functionality would trigger a fatal error if headers are
+				 * sent using an action is more forgiving.
+				 *
+				 * @since TBD
+				 */
+				do_action( 'tribe_admin_headers_about_to_be_sent' );
```

Comment:
We are using `tec_` as new prefixes now instead of `tribe_`

Reply:
Update to `tec_` thank you!